src/docs/codelabs/explicit-animations.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Explicit animations codelab"
+title: "Explicit animations"
```

Comment:
No need for quotation marks, I think. (Not that you added them. More an FYI.)

Reply:
You are quite right! And yes, 'twasn't me. I'll fix in a subsequent PR.